src/pybind/mgr/dashboard/frontend/src/app/shared/services/doc.service.ts

```diff
@@ -35,7 +35,10 @@ export class DocService {
       dashboard: `${domain}mgr/dashboard`,
       grafana: `${domain}mgr/dashboard/#enabling-the-embedding-of-grafana-dashboards`,
       orch: `${domain}mgr/orchestrator`,
-      pgs: `http://ceph.com/pgcalc`
+      pgs: `http://ceph.com/pgcalc`,
+      Status: `${domain}mgr/dashboard/#status`,
```

Comment:
Just spotted this now: these anchors are very generic. Could you please modify them to be a bit more specific, e.g. `#dashboard-landingpage-status`? You can include this documentation change in the same commit. Also, I wonder if it would be possible to create a test that performs a check if these URLs actually resolve or return a 404? This would help us to prevent any breakage if somebody updates the documentation without realizing that the dashboard expects these links to exist.

Reply:
For the record: we agreed on adding E2E tests that check for the existence of the linked documentation pages in a separate PR.